docs/dev_guide/diagnostic_ref/dpct1028.rst

```diff
@@ -27,3 +27,36 @@ Rewrite this code manually by using a supported random number generator. See the
 
 If the parameter is a CUDA device attribute:
 Redesign the code logic to remove use of the device attribute.
+
+For example, this original CUDA\* code:

```

Comment:
Can remove "\*" from CUDA here - already attributed earlier in page.

Reply:
done

Reply:
done